net/nali/Portfile

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/zu1k/nali 0.7.3 v
+revision            0
+
+license             MIT
+categories          net
+maintainers         {@tsln1998 gmail.com:tsln1998} openmaintainer
+
+description         An offline tool for querying IP geographic information
+long_description    ${description}
+homepage            https://github.com/zu1k/nali
```

Comment:
You don't need to specify `homepage` as that is automatically set by `go.setup`

Reply:
updated